### PR TITLE
Make URI values clickable

### PR DESCRIPTION
**Pull request by mdelez**
closes https://dasch.myjetbrains.com/youtrack/issue/DSP-529

uses js-lib v1.0.0-rc.9 and dsp-api v13.0.0-rc.12

### CODE REVIEW

**Comment by mdelez** [comment]
I also made a small change to disable showing an empty action bubble when hovering over a value when you don't have modify permission (on `/read` for example)

File: projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts [hunk 1]
```diff
@@ -254,7 +254,9 @@ export class DisplayEditComponent implements OnInit {
      * Show CRUD buttons and add 'highlighted' class to the element only if editModeActive is false
      */
     mouseEnter() {
-        this.showActionBubble = true;
+        if (this.canModify) {
```
Comment (tobiasschweizer) [inline projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts:257]: This is a general change, right?

File: projects/dsp-ui/src/lib/viewer/values/uri-value/uri-value.component.html [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
-    <p class="rm-value">{{valueFormControl.value}}</p>
+    <a target="_blank" href="{{valueFormControl.value}}">
```
Comment (tobiasschweizer) [inline projects/dsp-ui/src/lib/viewer/values/uri-value/uri-value.component.html:2]: Could the `<p>` element just contain the `<a>` tag?

https://stackoverflow.com/questions/13911274/can-a-p-tag-in-a-tag#:~:text=elements%20may%20not%20contain,This%20is%20not%20allowed.

File: projects/dsp-ui/package.json [hunk 3]
```diff
@@ -21,7 +21,7 @@
     "@angular/core": "^9.0.0",
     "@angular/material": "^9.0.0",
     "@angular/cdk": "^9.0.0",
-    "@dasch-swiss/dsp-js": "1.0.0-rc.7",
+    "@dasch-swiss/dsp-js": "1.0.0-rc.9",
```
Comment (tobiasschweizer) [inline projects/dsp-ui/package.json:24]: No changes in dsp-lib that could have an effect on dsp-ui?

File: projects/dsp-ui/src/lib/viewer/values/uri-value/uri-value.component.html (hunk 2 above)
Comment (mdelez) [reply projects/dsp-ui/src/lib/viewer/values/uri-value/uri-value.component.html]: fixed in d5350ec

File: projects/dsp-ui/package.json (hunk 3 above)
Comment (mdelez) [reply projects/dsp-ui/package.json:24]: I don't believe so. rc.8 added a mock onotology and rc.9 updated the DSP-API version

File: projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts (hunk 1 above)
Comment (mdelez) [reply projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts:257]: Can you clarify? :)

Comment (mdelez) [reply projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts:257]: oh, yeah, it's not really related to this PR but I figured it was a small enough change to sneak into this one. I can make a new PR if it's clearer.

File: projects/dsp-ui/src/lib/viewer/values/uri-value/uri-value.component.html (hunk 2 above)
Comment (tobiasschweizer) [reply projects/dsp-ui/src/lib/viewer/values/uri-value/uri-value.component.html]: thanks

**Review by tobiasschweizer — APPROVED** [review]
Thanks!

**Pull request merged**